source/blog/2021-11-24-release-notes.md

```diff
@@ -0,0 +1,104 @@
+    flag for the duration of the application. This improves performance but also makes them
+    not observe changes to the flag. The value of this flag typically does not change during
+    run time of an application, so this is deemed an acceptable tradeoff. [064e1535](https://github.com/twitter/scrooge/commit/064e1535ca50b4d7510e2d76fec8901b79a7d0e6)
+
+
+[Finagle](https://github.com/twitter/finagle/)
+==============================================
+
+Changed
+-------
+
+-   finagle-base-http: Promote several classes out of exp experimental package:
+    c.t.f.http.{GenStreamingSerialServerDispatcher, IdentityStreamTransport, StreamTransport} along
+    with internal support classes. [81169d53](https://github.com/twitter/finagle/commit/81169d5372442e5781fd1a62894eded080c0ac1d)
+
+Breaking API Changes
+--------------------
+
+-   finagle-core: Remove c.t.f.loadbalancer.Balancer.maxEffort. Remove the maxEffort
+    argument from Balancers.{p2c, p2cPeakEwma, aperture, aperturePeakEwmaUse, roundRobin}.
+    [25f01f77](https://github.com/twitter/finagle/commit/25f01f771d0f7cc513f64168aa20003791045776)
+-   finagle-core: c.t.f.tracing.ClientRequestTracingFilter has been removed.
+    Record relevant tracing information in your service or client directly. [bcd89491](https://github.com/twitter/finagle/commit/bcd89491ade6668c82143c0ea64ad09c68db12fa)
+-   finagle: Remove com.twitter.finagle.Group, and other rarely used and deprecated pieces that depend on it
+    com.twitter.finagle.memcached.TwitterCacheResolver, com.twitter.finagle.memcached.CacheNodeGroup,
+    com.twitter.finagle.memcached.RubyMemCacheClient, and com.twitter.finagle.memcached.PHPMemCacheClient.
+    Instead of Group, please use Var\[Set\[T\]\] or Activity\[Set\[T\]\] directly instead. [f6021319](https://github.com/twitter/finagle/commit/f60213199359efaf5492d461471e71058c76a1d9)
+
+Runtime Behavior Changes
+------------------------
+
+-   finagle: Update Caffeine cache library to version 2.9.2 [7c91f966](https://github.com/twitter/finagle/commit/7c91f966612dfc839824e144b08c80cfbf160532)
+
+
+[Finatra](https://github.com/twitter/finatra/)
+==============================================
+
+Added
+-----
+
+-   inject-core: Introduce a runAfterAll hook in c.t.inject.IntegrationTestMixin to allow for
+    running logic to clean-up test resources in the org.scalatest.BeforeAndAfterAll\#afterAll without
+    needing to 1) override org.scalatest.BeforeAndAfterAll\#afterAll, 2) ensure super is called for
+    other resources clean-up, and 3) ensure all resources get cleaned up, regardless of non-fatal
+    exceptions thrown as part of the clean-up logic and otherwise fail the TestSuite run.
+    [42c17b87](https://github.com/twitter/finatra/commit/42c17b87a8df306999d7fdd0ea73b85ebdcd4071)
+
+Changed
+-------
+
+-   finatra: Bump version of Jackson to 2.13.0 [044293fe](https://github.com/twitter/finatra/commit/044293fe14009791aa0e0c78edbaa7c8d1ac24da)
```

Comment:
I think we reverted the change, right? in the Nov release, the version should be 2.11.4? 
https://github.com/twitter/finatra/commit/61bc153be49f67db4fda9b092fe8918906a85c9f#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91

Reply:
Ahh interesting. It looks like it was never removed from the changelog. I will all of the changelogs.